### PR TITLE
Adds Global Eyebrow

### PR DESCRIPTION
**Pull request by anselmbradford**
## Additions

- Adds Global Eyebrow molecule.

## Testing

- Visit any page, resize page, note eyebrow at top of page and in mega menu at mobile sizes.

## Review

- @jimmynotjim 
- @sebworks 
- @KimberlyMunoz 

## Screenshots

Desktop size:
screen shot 2016-01-21 at 3 39 53 pm

Mobile size:
screen shot 2016-01-21 at 3 39 45 pm

### CODE REVIEW

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+<div class="m-global-eyebrow
+            m-global-eyebrow__{{ 'list' if value.is_in_list else 'horizontal' }}">
+    <div class="wrapper">
+        <div class="m-global-eyebrow_tagline">
+            An official website of the
+            <span class="m-global-eyebrow_tagline-usa">United States Government</span>
+        </div>
+        <ul class="m-global-eyebrow_languages">
+            <li>
+                <a href="/es/" hreflang="es">Español</a>
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: Can you also add `lang="es"` to this the appropriate lang tags to the others. I double checked and `hreflang` doesn't give any information to screenreaders, just search engines.

Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: Sure! Fixed in https://github.com/cfpb/cfgov-refresh/commit/767c68969f5e19c4502963bf32653e1b1141b40d

Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: Woo! Multi-language support! :us: :es: :cn:

**Comment by KimberlyMunoz** [comment]
:+1:

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+   value.is_in_list: Whether the molecule appears in a list or not.
+
+   ========================================================================== #}
+
+<div class="m-global-eyebrow
+            m-global-eyebrow__{{ 'list' if value.is_in_list else 'horizontal' }}">
+    <div class="wrapper">
+        <div class="m-global-eyebrow_tagline">
+            An official website of the
+            <span class="m-global-eyebrow_tagline-usa">United States Government</span>
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: USA :us: USA :us:

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+   Creates markup for Global Eyebrow molecule.
+
+   value.is_in_list: Whether the molecule appears in a list or not.
+
+   ========================================================================== #}
+
+<div class="m-global-eyebrow
+            m-global-eyebrow__{{ 'list' if value.is_in_list else 'horizontal' }}">
+    <div class="wrapper">
+        <div class="m-global-eyebrow_tagline">
```
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: Should this semantically be a `p`?

File: cfgov/unprocessed/css/brand-palette.less [hunk 4]
```diff
@@ -24,6 +24,7 @@
 @gray:              #75787B;
 @gray-80:           #919395;
 @gray-50:           #BABBBD;
+@gray-40:           #b4b5b6; // Close to what was known as "Gray 50"
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/brand-palette.less]: Should the spec be updated to the new palette rather than adding new vars?

File: cfgov/unprocessed/css/molecules/global-eyebrow.less [hunk 5]
```diff
@@ -0,0 +1,105 @@
+/* topdoc
+  name: Social Media
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Update name and markup example from social media

File: cfgov/unprocessed/css/molecules/global-eyebrow.less [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+              li.list_item
+                .m-social-media_icon
+                  span.cf-icon
+                  span.u-visually-hidden
+  tags:
+    - cfgov-molecules
+*/
+
+.m-global-eyebrow {
+    .webfont-regular();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Might leave this out since I've opened #1360

File: cfgov/unprocessed/css/molecules/global-eyebrow.less [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+        });
+
+        &-usa {
+            white-space: nowrap;
+        }
+    }
+
+    &_languages {
+        .list__horizontal();
+        .list__unstyled();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Any reason not to use these in the markup and avoid duplication in the compiled css?

File: cfgov/unprocessed/css/molecules/global-eyebrow.less [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+        li {
+            .list_item();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Same question with this class

File: cfgov/unprocessed/css/molecules/global-eyebrow.less [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+            margin-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+            margin-bottom: 0;
+
+            &:last-child {
+                margin-right: 0;
+            }
+        }
+
+        a {
+            .u-link__no-border();
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: And this one

Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: This one is a mixin https://cfpb.github.io/cf-core/docs/#link-mixins

File: cfgov/unprocessed/css/molecules/global-eyebrow.less (hunk 7 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: These are using properties of an unstyled horizontal list (horizontal layout, no bullets), but are using different padding, so it seems misleading to have it in the markup when it differs from what the CF docs show for unstyled, horizontal lists.

File: cfgov/unprocessed/css/molecules/global-eyebrow.less (hunk 5 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Oops! Fixed in https://github.com/cfpb/cfgov-refresh/commit/34a80bf2ca7ea5659b89e959eb73c88ecf91b1b0

File: cfgov/unprocessed/css/brand-palette.less (hunk 4 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/brand-palette.less]: This would require wide-ranging updates across the codebase to adjust `@gray-50` to `@gray-40`,  which I think would be best to deliver in its own PR. At that time this whole file can just be overwritten by what's in the cf generator.

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html (hunk 3 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: So I had it a `p` and it was adding a bottom margin. Instead of overriding that I just made it a `div`. Does that make sense?

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html (hunk 2 above)
Comment (anselmbradford) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: :bowtie:

File: cfgov/unprocessed/css/molecules/global-eyebrow.less (hunk 7 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: We've done that elsewhere though, where we use `block` and it's modifiers and then make adjustments to the elem that counter the `block` styles. I think you're ok doing the same here.

File: cfgov/unprocessed/css/molecules/global-eyebrow.less (hunk 9 above)
Comment (jimmynotjim) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Whoops, I thought all our mixin-only ones had params. Carry on :D

File: cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html (hunk 3 above)
Comment (jimmynotjim) [inline cfgov/jinja2/v1/_includes/molecules/global-eyebrow.html]: Makes sense, thanks.

File: cfgov/unprocessed/css/molecules/global-eyebrow.less (hunk 6 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Fixed in https://github.com/cfpb/cfgov-refresh/commit/ed2b42ad78a1b8c384073a0a321387ec2be2587a

File: cfgov/unprocessed/css/molecules/global-eyebrow.less (hunk 7 above)
Comment (anselmbradford) [inline cfgov/unprocessed/css/molecules/global-eyebrow.less]: Removed in https://github.com/cfpb/cfgov-refresh/commit/db6b4fbcd119694545fc137fe70543725ac13f5e

**Comment by anselmbradford** [comment]
Ready for review again

**Comment by sebworks** [comment]
:+1:

**Comment by jimmynotjim** [comment]
:+1:
